src/app/source-file-tabs/source-file-tabs.component.html

```diff
@@ -1,6 +1,6 @@
 <app-loading class="w-100" [loading]="loading">
-  <div *ngIf="version" class="p-3">
-    <mat-card appearance="outlined" class="alert alert-warning" *ngIf="validationMessage">
+  <div *ngIf="version && validationMessage" class="p-3">
```

Comment:
Noticed that this extra padding was always being applied. Changed it so it only applies if there's validation messages, which is similar to the other mat-card alerts below